content/specs/handle.md

```diff
@@ -92,11 +92,68 @@ laptop.local
 blah.arpa
 ```
 
-### Usage and Implementation Guidelines
+## Handle Resolution
```

Comment:
Thanks! This is all great.

As mentioned before, definitely questions here around how this interacts with federation, but that's for the future. And ideally guidance on when/how to deal with handles in records as alternatives to DIDs.